Library/Homebrew/test/Gemfile

```diff
@@ -1,9 +1,15 @@
 source "https://rubygems.org"
 
-gem "mocha", "~> 1.1"
-gem "minitest", "~> 5.3"
-gem "rake", "~> 10.3"
-gem "parallel_tests", "~> 2.9"
+group :test do
+  gem "minitest", "~> 5.9"
+  gem "minitest-reporters"
+  gem "mocha", "~> 1.2"
+  gem "rake", "~> 10.5"
```

Comment:
Given a bunch of these are being bumped let's just remove all the version pinning here (we get it from `.lock` anyway) and restrict it to the cases where the newest version breaks `brew tests`